### PR TITLE
Fix broken links (wrong relative link, wrong anchor format)

### PR DESCRIPTION
**Pull request by robertjd**
Find and replace operation to fix links that:
- Were not using full paths
- Were not putting trailing slashes before an anchor

@jakubvul and @bretterer to sanity check these changes

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/api/resources/apps/index.md [hunk 1]
```diff
@@ -3564,7 +3564,7 @@ Assigns a group to an application
 | Parameter     | Description                                     | Param Type | DataType                                      | Required | Default |
 | ---------     | ----------------------------------------------- | ---------- | --------------------------------------------- | -------- | ------- |
 | applicationId | `id` of an [app](#application-model)            | URL        | String                                        | TRUE     |         |
-| groupId       | unique key of a valid [Group](groups)           | URL        | String                                        | TRUE     |         |
+| groupId       | unique key of a valid [Group](/docs/api/resources/groups)           | URL        | String                                        | TRUE     |         |
```
Comment (swiftone) [inline packages/@okta/vuepress-site/docs/api/resources/apps/index.md]: Shouldn't this be `/docs/api/resources/groups/` (w/trailing slash)?  The site corrects it, but...

File: packages/@okta/vuepress-site/docs/api/resources/idps/index.md [hunk 2]
```diff
@@ -5774,7 +5774,7 @@ All IdP transactions have the following properties:
 | status        | status of transaction                                                                  | `ACCOUNT_JIT`, `ACCOUNT_LINK` or `SUCCESS`                      | FALSE     | FALSE    | TRUE     |
 | created       | timestamp when transaction was created                                                 | Date                                                            | FALSE     | FALSE    | TRUE     |
 | expiresAt     | timestamp when transaction expires                                                     | Date                                                            | FALSE     | FALSE    | TRUE     |
-| sessionToken  | ephemeral [one-time token](authn#session-token) used to bootstrap an Okta session      | String                                                          | TRUE      | FALSE    | TRUE     |
+| sessionToken  | ephemeral [one-time token](/docs/api/resources/authn/#session-token) used to bootstrap an Okta session      | String                                                          | TRUE      | FALSE    | TRUE     |
```
Comment (robertjd) [inline packages/@okta/vuepress-site/docs/api/resources/idps/index.md]: @jakubvul where is this supposed to go?

File: packages/@okta/vuepress-site/docs/api/resources/apps/index.md (hunk 1 above)
Comment (robertjd) [reply packages/@okta/vuepress-site/docs/api/resources/apps/index.md]: I don't have an opinion on this one.  We could lint it, but that feels a bit over-bearing.

Comment (bretterer) [reply packages/@okta/vuepress-site/docs/api/resources/apps/index.md]: When we get a good winter set up, I'll add this and fix everything that's broken

Comment (bretterer) [reply packages/@okta/vuepress-site/docs/api/resources/apps/index.md]: By broken, I mean slash missing

**Pull request merged**